content/post/74-tunneling-http-in-ipfs.md

```diff
@@ -0,0 +1,63 @@
+---
+date: 2019-03-08
+url: 74-tunneling-http-in-ipfs
+tags: libp2p, go-ipfs
+title: Securely tunneling HTTP through IPFS with libp2p
+author: Hector Sanjuan
+---
+
+The last [go-ipfs](https://github.com/ipfs/go-ipfs) release (0.4.19) includes
+[a new experimental feature](https://github.com/ipfs/go-ipfs/blob/master/docs/experimental-features.md#p2p-http-proxy)
+which allows any peer to serve an HTTP endpoint through the IPFS daemon's
+[libp2p host](https://libp2p.io) and any other peer to expose it under the
+`/p2p/<peer-id>/<web-url>` path in the gateway endpoint. It was a contribution
+by [Peergos]( https://peergos.org/) and it means that the web pages and web
```

Comment:
@ianopolous thank you! Wanna help to review this post as well?

Reply:
@daviddias Sure, no problem